src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -258,7 +258,7 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin) {
       // Administrative Dissolution, Put Back On, Freeze/Unfreeze Business require a detailed comment
       (v: string) => (
         !!v ||
-        (!this.isAdministrativeDissolution && !this.isPutBackOn && !this.isAdministerFreeze) ||
```

Comment:
There's no component, and thus no rules, for admin freeze. Ditto line 274 below.